common/tier4_localization_rviz_plugin/src/pose_history/pose_history_display.hpp

```diff
@@ -47,23 +44,24 @@ class PoseHistory : public rviz_common::MessageFilterDisplay<geometry_msgs::msg:
 public:
   PoseHistory();
   ~PoseHistory() override;
+  PoseHistory(const PoseHistory &) = delete;
+  PoseHistory(const PoseHistory &&) = delete;
+  PoseHistory & operator=(const PoseHistory &) = delete;
+  PoseHistory & operator=(const PoseHistory &&) = delete;
 
 protected:
   void onInitialize() override;
   void onEnable() override;
   void onDisable() override;
   void update(float wall_dt, float ros_dt) override;
 
-private Q_SLOTS:
-  void subscribe();
-  void unsubscribe();
-  void processMessage(const geometry_msgs::msg::PoseStamped::ConstSharedPtr message);
-
-private:  // NOLINT for Qt
-  void updateHistory();
-  void updateLines();
-
 private:
+  void subscribe() override;
+  void unsubscribe() override;
+  void processMessage(const geometry_msgs::msg::PoseStamped::ConstSharedPtr message) override;
```

Comment:
@h-ohta 
[IMO]
I think every function should be lowerCamelCase or snake_case if you want to change this.
other wise this change is worse than before.

Reply:
These function are defined in parent class. So I cant change it

Reply:
I understand but it looks like strange code style with QT :thinking: 
if you like this I think it's ok.

Reply:
I resolve this